src/api.rs

```diff
@@ -25,7 +25,7 @@ pub struct BlockData<T: Serialize> {
     /// Header for block
     pub header: BlockHeader,
     /// Data for block
-    #[serde(skip_serializing_if = "Option::is_none")]
+    //#[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
Was this change intentional?

Reply:
Yes, that looks to be one of the pieces that is required for deterministic/single events.

Reply:
got it, maybe nice to delete but all good